index.css

```diff
@@ -221,18 +250,24 @@
 .article2 button,
 .article3 button,
 .article4 button,
-.about_self_section button {
+.about_self_section button,
+.btn_1 {
   padding: 10px 10px;
   border: 1px solid #6070ff;
   border-radius: 10px;
   background-color: #fff;
 }
 
+.btn_1 {
+  color: #6070ff;
+}
```

Comment:
- [ ] [OPTIONAL] I think it will be a good idea to put `cursor: pointer;` on this button. It will be a great experience for users when they hover over it. 😎️